src/main/scala/com/virtuslab/shared_indexes/core/IntelliJ.scala

```diff
@@ -35,11 +40,35 @@ class IntelliJ(
     val process = new ProcessBuilder()
       .directory(workingDir.toIO)
       .command(command: _*)
-      .inheritIO()
+      .redirectOutput(ProcessBuilder.Redirect.PIPE)
+      .redirectError(ProcessBuilder.Redirect.PIPE)
       .start()
 
+    def streamOutput(inputStream: java.io.InputStream, isError: Boolean): Future[Unit] = Future {
+      val reader = new BufferedReader(new InputStreamReader(inputStream))
+      var line: String = null
+      val isStatusEnabled = !isError && !logger.isDebugEnabled()
+      val taskStatus = TaskStatus("Generating shared indexes", isStatusEnabled)
+      taskStatus.start()
+      while ({ line = reader.readLine(); line != null }) {
+        if (isError)
+          logger.debug(line)
+        else
+          logger.debug(line)
+          taskStatus.updateSpinner(line)
```

Comment:
Looks like you have missed the braces around the else clause.
```suggestion
        else {
          logger.debug(line)
          taskStatus.updateSpinner(line)
        }
```